### PR TITLE
IMF Mechanic

### PR DESCRIPTION
**Pull request by Arcadyi**

### CODE REVIEW

File: localization/english/CWP/CWP_imf_l_english.yml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+je_imf_concessionary_loan_goal: "[GetPlayer.GetName] has exceeded the allotted payment time - [GetPlayer.MakeScope.ScriptValue('imf_loan_payment_years_left')|v1] Years Left"
```
Comment (JackEPepper) [inline localization/english/CWP/CWP_imf_l_english.yml:89]: Redundant localization error

File: localization/english/CWP/CWP_imf_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+je_imf_nonconcessionary_loan_goal: "[GetPlayer.GetName] has exceeded the allotted payment time - [GetPlayer.MakeScope.ScriptValue('imf_loan_payment_years_left')|v1] Years Left"
```
Comment (JackEPepper) [inline localization/english/CWP/CWP_imf_l_english.yml:91]: Same as above, redundant loc

File: common/script_values/imf_values.txt [hunk 3]
```diff
@@ -1 +1,818 @@
-﻿
\ No newline at end of file
+﻿global_gdp = {
```
Comment (JackEPepper) [inline common/script_values/imf_values.txt:1]: Suggest rename, current name produces error on startup and will overshadow the event_target of the same name

**Review by DZappala — CHANGES_REQUESTED** [review]

**Review by JackEPepper — APPROVED** [review]
left a few comments, minor stuff mostly

File: localization/english/CWP/CWP_concepts_l_english.yml [hunk 4]
```diff
@@ -136,5 +136,20 @@
...
+
+  concept_imf: "International Monetary Fund"
+  concept_imf_desc: "[concept_imf] (IMF) is an [concept_international_organization] established in 1944 with the goal of promoting global economic stability and growth. It provides financial assistance, policy advice, and technical expertise to its member countries to address economic challenges and imbalances. The IMF plays a crucial role in managing financial crises, promoting exchange rate stability, and supporting sustainable economic development worldwide."
+  
+  concept_external_debt: "External Debt"
+  concept_external_debt_desc: "TODO"
+  
+  concept_imf_bailout: "Bailout"
+  concept_imf_bailout_desc: "TODO"
+
```
Comment (raxo2222) [inline localization/english/CWP/CWP_concepts_l_english.yml:154]: Those two TODO entries need proper desc

File: common/scripted_effects/cwp_onaction_imf_ai.txt [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+				has_modifier = imf_austerity_measures
+				has_journal_entry = je_imf_austerity_measures
+				has_journal_entry = je_imf_bailout_loan	
+			}
+		}
+
+		random = {
+			chance = 10
+			modifier = {
+				add = imf_ai_wants_loan
```
[the remark this answers is not in the document]
Comment (Arcadyi) [reply common/scripted_effects/cwp_onaction_imf_ai.txt:49]: i guess its possible that it doesn't accept negative values, will check

**Pull request merged**